xla/stream_executor/stream_executor_internal.h

```diff
@@ -250,6 +256,10 @@ class StreamExecutorInterface {
     return absl::UnimplementedError("Not Implemented");
   }
 
+  virtual tsl::Status MemallocOnDevice(uint64_t size, DeviceMemoryBase* dst) {
```

Comment:
Didn't find any overrides, I don't think it's needed for StreamExecutor, as it's just a regular `Allocate`

Reply:
Removed this.